smt/utils/design_space.py

```diff
@@ -742,6 +744,11 @@ def _is_num(val):
                     cs_vars[name] = CategoricalHyperparameter(name, choices=dv.values)
                 else:
                     raise ValueError(f"Unknown variable type: {dv!r}")
+            seed = None
```

Comment:
I suggest to make a utility function `to_seed()` as these lines are repeated twice later on.